vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -66,11 +66,13 @@ General.run=Run
 Dialogs.disable=Don't show this again
 Dialogs.show_details=Show details
 Dialogs.hide_details=Hide details
+Dialogs.dont_show_again=Don't shpow this again
 Dialogs.out_of_date=Out Of Date Custom Code
 Dialogs.deprecated.later=some later date
-Dialogs.deprecated.message=This module contains custom code that is calling the VASSAL function %1$s. This function is scheduled to be removed from VASSAL after %2$s and this module will cease to function.
-Dialogs.check_for_updated_module=Please check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed.
-Dialogs.out_dated_usage=This module contains custom code that is calling into VASSAL in a way that is no longer recommended (%1$s). This will result in poor performance and/or bugs.
+Dialogs.deprecated.message=This module contains outdated custom code, which is not guaranteed to work with versions of VASSAL released after %1$s.
+Dialogs.check_for_updated_module=Please check whether there is an updated version of this module. If not, please contact the maintainer of this module and request that it be fixed, or post a message on the Vassal forums and ask for help.
+Dialogs.deprecated.detailed_message=Deprecated VASSAL function %1$s called from custom code method %2$s. This VASSAL method is scheduled to be removed after %3$s.
+Dialogs.out_dated_usage=This module contains custom code that is calling into VASSAL in a way that is no longer recommended (%1$s). This may result in poor performance and/or bugs.
```

Comment:
Typos fixed

Reply:
No, I meant that `outdated` had a space somewhere else and shouldn't. :)

Reply:
In the key name: `Dialogs.out_dated_usage` --> `Dialogs.outdated_usage`

Reply:
Typo is fixed. I would rather not change that key as it has already been translated. It's not visible to anyone except the translators and they work off the English text anyway.

Reply:
I don't understand---how does they key already having been translated make a difference? Wouldn't we just change the name of the key to match in all the translations?

Reply:
Yes, If we where still controlling that. But since it goes via Crowdin, it essentially wipes out the existing translation and gives them a new Key and new text so they have to do the translation again. Many of the keys are contractions or have typos, they don't need to be perfect, just unique.

Reply:
I'm not sure if Crowdin reads the non-english translations if we make changes to them, I thought it was one way English -> Crowdin -> _xx translations.